Organization/Organization/MainViewController.m

```diff
@@ -1,57 +1,67 @@
 #import "MainViewController.h"
 #import "DetailViewController.h"
 #import "CreateEmployeeViewController.h"
-#import "Employee.h"
-#import "Organization.h"
+#import "Employee+CoreDataClass.h"
+#import "Organization+CoreDataClass.h"
+#import "AppDelegate.h"
 
-@interface MainViewController ()		
 
-@end
 
 @implementation MainViewController
 
+
+
+NSString* attributeValue = @"Siemens";
+NSString* attributeName = @"name"; //genious constant
+AppDelegate *delegate;
+
 - (void)viewDidLoad
 {
     [super viewDidLoad];
+    delegate = (AppDelegate*)[[UIApplication sharedApplication] delegate];
+ 
+   
+    NSFetchRequest *request = [[NSFetchRequest alloc] init];
+    NSEntityDescription *entity = [NSEntityDescription entityForName:@"Organization" inManagedObjectContext:delegate.managedObjectContext];
+    [request setEntity:entity];
+    self.org = [delegate.managedObjectContext executeFetchRequest:request error:nil] == nil ?
```

Comment:
Too complicated line, execute fetch request should be performed only once. In Objective-C check for null as `if (obj)`, not `if (obj != nil)`